### PR TITLE
Fixes scrolling in compound and geneset page.

### PR DESCRIPTION
**Pull request by andricDu**
Opening this for some testing.

### CODE REVIEW

File: dcc-portal-ui/app/scripts/compounds/js/compounds.js [hunk 1]
```diff
@@ -61,13 +62,28 @@ angular.module('icgc.compounds.controllers', ['icgc.compounds.services'])
       return promise;
     }
 
+    function fixScroll() {
+      var current = $location.hash();
+
+      // If we are linked to the page and we have a hash, wait for render to finish and scroll to position. 
+      $timeout(function () {
```
Comment (btiernay) [inline dcc-portal-ui/app/scripts/compounds/js/compounds.js]: Maybe this should be a callback on an application event. Maybe from `Page`? Dunno

Comment (andricDu) [inline dcc-portal-ui/app/scripts/compounds/js/compounds.js]: Unfortunately I've seen this as the standard practice for executing code post render in angular. 

Backbone has a handy `afterRender` callback that every view has, but there is no analogue in angular.

Comment (btiernay) [inline dcc-portal-ui/app/scripts/compounds/js/compounds.js]: The problem is it needs to happen after all the ajax have settled. Right now it just ignores that information. So you need application level support to do this properly.

**Comment by btiernay** [comment]
Good enough for now!

**Comment by btiernay** [comment]
Please update release notes.

**Comment by andricDu** [comment]
Done!

**Pull request merged**